src/Input/TextField/TextField.tsx

```diff
@@ -112,6 +112,7 @@ class TextField extends React.Component<Props, State> {
         />
         {!removeFloatingLabel && (
           <TextFieldLabel
+            data-testid="textfield-label"
```

Comment:
was thinking is it better to set a parameter for this 🤔 ?

Reply:
Oh what do you mean by set a parameter?

Reply:
make create a prop for this?